README.md

```diff
@@ -27,10 +27,10 @@ Library](https://briansmith.org/insanity-pkix).
 The Rust compiler statically guarantees there are no buffer overflows,
 uses-after-free, double-frees, data races, etc. in webpki. webpki takes
 advantage of Rust's borrow checker to ensure that its **zero-copy parsing**
-strategy is safe and efficient. webpki *never* allocates memory on the heap,
-and it maintains a tight bound on the amount of stack memory it uses. webpki
-avoids all superfluous PKIX features in order to keep its object code size
-small. Further reducing the code size of webpki is an important goal.
+strategy is safe and efficient. By default, webpki *never* allocates memory on
```

Comment:
"By default, .. never" reads a bit odd. Can we be more specific?

Reply:
Yeah -- I imagine the intention might have been more like "certificate verification doesn't need to allocate memory".

Reply:
Yeah, those sound good to me!

Reply:
I've added a more opinionated commit to this branch that I think achieves some of what we discussed above (and side-steps any requirement to talk about low-level implementation details like allocation):

https://github.com/rustls/webpki/pull/78/commits/0bcd0a8cf89d9cad063e0c6c7eda3336e458f2da

I tried to keep the descriptions of what webpki does accurate, short in summary, and not _too_ heavy on obscure technical terms (DER, ASN.1, X509, etc). Some of the descriptions still use terms like "path building" but it's challenging to summarize these concepts and I think it's likely someone digging past the Rustls crate and into Webpki has a little bit of domain knowledge.

Let me know what you think. I'm happy to iterate further.